pkg/nfd-topology-updater/nfd-topology-updater.go

```diff
@@ -172,7 +172,7 @@ func (w *nfdTopologyUpdater) Stop() {
 	}
 }
 
-func (w *nfdTopologyUpdater) updateNodeResourceTopology(zoneInfo v1alpha1.ZoneList) error {
+func (w *nfdTopologyUpdater) updateNodeResourceTopology(zoneInfo v1alpha1.ZoneList, annotations map[string]string) error {
```

Comment:
we shoudl evaluate here to wrap the args in a `struct`

Reply:
I have changed it to use `ScanResponse` 
I could have created a new struct including all the params ( right now only ZoneList) but as this is a "private" method that is called only once it does not seems worthy.